src/tss2-fapi/ifapi_helpers.c

```diff
@@ -922,12 +922,51 @@ append_object_to_list(void *object, NODE_OBJECT_T **object_list)
     return TSS2_RC_SUCCESS;
 }
 
+/**  Compute the name of a hierarchy object.
+ *
+ * The TPM will be computed from the esys handle and the name
```

Comment:
This sentence appears werid

Reply:
@AndreasFuchsSIT Indeed, I have changed it.